vm/chef/cookbooks/redmine/recipes/default.rb

```diff
@@ -18,61 +18,195 @@
 include_recipe 'apache2::mod-passenger'
 include_recipe 'apache2::security-config'
 include_recipe 'mysql'
+include_recipe 'rvm'
 
-# Configure redmine package (preventing interactive dialogs to show up):
-bash 'configure_apache_site' do
+apt_update 'update' do
+  action :update
+end
+
+package 'Install Packages' do
+  package_name node['redmine']['packages']
+  action :install
+end
+
+user node['redmine']['user'] do
+  action :create
+  home "/home/#{node['redmine']['user']}"
+  password node['redmine']['password']
+  shell '/bin/bash'
```

Comment:
/sbin/nologin

Reply:
Accepted. It will be included in next PR.

Reply:
Fixed.

Reply:
Fixed.